File: 62/unique-paths.c
int uniquePaths(int m, int n){
    int table[101][101];
    for(int j=0; j<n; j++) table[0][j] = 1;

    for(int i=1; i<m; i++){
        table[i][0] = table[i-1][0];
        for(int j=1; j<n; j++){
            table[i][j] = table[i-1][j] + table[i][j-1];
        }
    }
    return table[m-1][n-1];
}

File: 62/unique-paths.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        table = [[0]*n for _ in range(m)]
        
        for i in range(m):
            for j in range(n):
                if i==0 or j==0 : table[i][j] = 1;
                else:
                    table[i][j] = table[i-1][j] + table[i][j-1]

        return table[m-1][n-1]
